tests/validator/is-dateTime.spec.ts

```diff
@@ -0,0 +1,186 @@
+import {STRule} from '../../src/rule/rule';
+import {STRuleModifiers} from '../../src/rule/modifiers';
+import {makeIsDateTime} from '../../src/validator/is-dateTime';
+
```

Comment:
Per our conversation, update `is-dateTime.spec.ts` to be `is-date-time.spec.ts` for project consistency.